subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -163,21 +164,23 @@ int lll_init(void)
 	hal_swi_init();
 
 	/* Connect ISRs */
-	IRQ_DIRECT_CONNECT(RADIO_IRQn, CONFIG_BT_CTLR_LLL_PRIO,
-			   radio_nrf5_isr, IRQ_CONNECT_FLAGS);
+	NRF_IRQ_DIRECT_DYNAMIC(RADIO_IRQn, CONFIG_BT_CTLR_LLL_PRIO,
+			       IRQ_CONNECT_FLAGS);
```

Comment:
Should CONFIG_BT_LLL_VENDOR_NORDIC select CONFIG_DYNAMIC_INTERRUPTS ?

Then apps wouldn't have to turn it on.

Also, a Kconfig source for CONFIG_DYNAMIC_INTERRUPTS is missing.

Perhaps it should be promptless as well ...

Reply:
Will do this in a different way. Just hard-switched it to have an example in this PR.